go/pkg/rexec/rexec_test.go

```diff
@@ -441,6 +441,58 @@ func TestGetOutputFileDigests(t *testing.T) {
 	}
 }
 
+func TestGetOutputDigests(t *testing.T) {
+	e, cleanup := fakes.NewTestEnv(t)
+	defer cleanup()
+	fooPath := filepath.Join(e.ExecRoot, "foo")
+	fooBlob := []byte("hello")
+	if err := ioutil.WriteFile(fooPath, fooBlob, 0777); err != nil {
+		t.Fatalf("failed to write input file %s", fooBlob)
+	}
+	cmd := &command.Command{
+		Args:        []string{"tool"},
+		ExecRoot:    e.ExecRoot,
+		InputSpec:   &command.InputSpec{Inputs: []string{"foo"}},
+		OutputFiles: []string{"a/b/out"},
```

Comment:
Can you expand the tests cases to add few more files/dirs combinations?

Reply:
Added additional directories and file to tests.

a/ : contains subdir
a/b/: contains file + subdir
a/b/c/: contains file